skeleton/webpack/webpack.config.js

```diff
@@ -245,7 +259,10 @@ module.exports = ({ production, server, extractCss, coverage, analyze, karma } =
       // @if feat.sass
       {
         test: /\.scss$/,
-        use: ['style-loader', 'css-loader', 'sass-loader'],
+        use: extractCss ? [{
+          loader: MiniCssExtractPlugin.loader
+        }, ...sassRules
+        ]: ['style-loader', ...sassRules],
```

Comment:
Your newly created sassRules bypassed postcss config.

It looks like it could be
```
}, 'css-loader', 'sass-loader'
]: ['style-loader', ...cssRules, 'sass-loader']
```


Or
```
}, 'css-loader', ...sassRules
]: ['style-loader', ...cssRules, ...sassRules]
```
With
```
const sassRules = [
	  {
	     loader: "sass-loader", 
	     options: {
	       includePaths: ["node_modules"]
	     }
	  }
	];
```

Also, to be complete, need to do the same thing for less and stylus config.
I am no master of webpack, I am not sure why postcss is skipped when extractCss is on, I mean in our existing config.

Reply:
I did not foresee that someone wanted to use SASS and PostCSS together.
Normally someone uses one or the other. But I might be wrong of course.

Reply:
postcss is not a preprocessor, it's a post-processor, it can be used with css or the css result compiled from less/sass/stylus.

For example the autoprefixer (plugin for postcss) is a useful step for any css setup. Although I am not sure if webpack does some browser prefix automatically.

Reply:
It is useful for minification (via nanocss). So I updated the files.